cards/src/main/resources/cards/custom/group0/other_death_knight/tgt/spell_midnight_ride.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Midnight Ride",
+  "baseManaCost": 4,
+  "type": "SPELL",
+  "heroClass": "SPIRIT",
+  "rarity": "EPIC",
+  "description": "Shuffle all minions into their owner's deck. They are summoned when drawn.",
+  "targetSelection": "NONE",
+  "spell": {
+    "class": "MetaSpell",
+    "spells": [
+      {
+        "class": "AddEnchantmentSpell",
+        "target": "FRIENDLY_PLAYER",
+        "card": "enchantment_midnight_ride",
```

Comment:
Ah, I see. What if you added this enchantment directly to the cards, and the enchantment was a `passiveTrigger` instead?